Tests/Common/ZipFileChecker.cs

```diff
@@ -77,24 +72,29 @@ public void AssertZipDoesNotContainFiles(params string[] unexpectedRelativePaths
             {
                 testContext.WriteLine("ZipFileChecker: checking for file '{0}'", relativePath);
 
-                string[] matchingFiles = Directory.GetFiles(UnzippedDirectoryPath, relativePath, SearchOption.TopDirectoryOnly);
+                string[] matchingFiles = Directory.GetFiles(unzippedDirectoryPath, relativePath, SearchOption.TopDirectoryOnly);
 
                 matchingFiles.Length.Should().Be(0, $"Zip file contains unexpected file: {relativePath}");
             }
         }
 
+        public void AssertZipContainsOnlyExpectedEntries(params string[] expectedEntriesName)
+        {
+            archiveFile.Entries.Select(x => x.FullName).Should().HaveSameCount(expectedEntriesName).And.Contain(expectedEntriesName);
```

Comment:
This should do the trick in one go, without the count check. 
`.Should().BeEquivalentTo()`

Reply:
This can be arrow method